jade/page-contents/forms_content.html

```diff
@@ -911,6 +911,29 @@ <h2 class="header">Autocomplete</h2>
     minLength: 1, // The minimum length of the input for the autocomplete to start. Default: 1.
   });
         </code></pre>
+
+        <h4>Remote ajax call</h4>
+        <p>To fetch a remote server, you need to replace the "data" parameter with the "url" parameter containing the remote content url.</p>
+        <p>Few things to note :</p>
+        <ul>
+          <li>The request is send by GET according to HTML protocol spec for fetching content</li>
+          <li>The get variable name containing the search is "val" </li>
+          <li>Your server must respond in XML or JSON</li>
+          <li>The output should be an object's array. Each object must have at least a "value" key containing the text. It can also have an "img" key containing the optional picture url. <b>Any other key present in the response objects can be retrieve in the onAutocomplete callback. See below :</b></li>
```

Comment:
I think we should display an example of the result you expect instead of explanation.

Reply:
I totally agree, but for this, it takes to have an api endpoint to fetch data to autocomplete.

Reply:
I was talking about writing an exemple in this text like this : 

> Here's an example of what we expect of `http://your-awesome-api.com?val=google` : 
> 
> ```json
> {"result" : [
>   {
>      "value":"google-map",
>      "img":"url-to-google-map.png"
>   },
>   {
>      "value":"google-doc",
>      "img":"url-to-google-doc.png"
>   },
>   ...
> ]}
> ```

Reply:
You're right. I will update the doc this evening.